### PR TITLE
Fix issue #2

### PR DESCRIPTION
**Pull request by jeffslofish**

### CODE REVIEW

**Comment by jeffslofish** [comment]
I found your project on http://issuehub.io/

**Comment by fiatjaf** [comment]
Please tell what you did!

File: jq.js [hunk 1]
```diff
@@ -2648,7 +2648,8 @@ function copyTempDouble(ptr) {
       }};
   
   var NODEFS={isWindows:false,staticInit:function () {
-        NODEFS.isWindows = !!process.platform.match(/^win/);
+        var os = require('os');
```
Comment (jeffslofish) [inline jq.js]: Weird, I only changed these two lines. I don't know why all the other changes are showing up.

**Comment by jeffslofish** [comment]
The only change I had intended to make was this:

````
        var os = require('os');
        NODEFS.isWindows = !!os.platform().match(/^win/);

````

I am not sure why all the other changes showed up. =(

**Comment by fiatjaf** [comment]
I understand it, but that doesn't quite solve it, because jq.js is a file generated by emscripten, so doing this adds a manual edit to jq.js to the build process, which is a quite bad thing.

But you nonetheless gave me and idea that might work.

**Pull request closed without merging**